dts/common/nordic/nrf54h20.dtsi

```diff
@@ -820,6 +855,7 @@
 				compatible = "nordic,nrf-dppic-global";
 				reg = <0x981000 0x1000>;
 				status = "disabled";
+				power-domains = <&gpd NRF_GPD_SLOW_MAIN>;
```

Comment:
Shouldn't DPPIC131 be in `NRF_GPD_SLOW_ACTIVE`, like SAADC, [LP]COMP, TEMP, etc.?

Reply:
yes, good catch